WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -410,9 +428,10 @@ private extension DashboardViewController {
 private extension DashboardViewController {
     func reloadData(forced: Bool) {
         DDLogInfo("♻️ Requesting dashboard data be reloaded...")
-        dashboardUI?.reloadData(forced: forced, completion: { [weak self] in
-            self?.configureTitle()
-        })
+        Task {
```

Comment:
By default, a task executes on an automatically managed background thread, and since we are updating UI perhaps this is not what we want here 😄 
You can also mark your task’s closure as being @MainActor, to be sure that this code is executed on the main thread, like this:

```
Task { @MainActor in
            await dashboardUI?.reloadData(forced: forced)
            configureTitle()
 }
```